server/framework.lua

```diff
@@ -24,6 +24,10 @@ local function hasItem(player, items)
 	end
 end
 
+local function getCharacterId(player)
+	return tostring(player.charid)
+end
```

Comment:
In QB the character id isn't a number, so to be compatible the character id list needs to be handled as a string.

Reply:
I don't know about parseInt, but with Lua you can just do `tonumber(v) or v`. Store it as the appropriate type rather than needing to constantly convert during runtime.